{
  "id": "oval:org.debian:def:215521045210460961785260137743859271887",
  "class": "vulnerability",
  "metadata": {
    "title": "CVE-2018-20225 python-pip",
    "affected": {
      "family": "unix",
      "platform": "Debian GNU/Linux 11",
      "product": "python-pip"
    },
    "reference": [
      {
        "source": "CVE",
        "ref_id": "CVE-2018-20225",
        "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20225"
      }
    ],
    "description": "An issue was discovered in pip (all versions) because it installs the version with the highest version number, even if the user had intended to obtain a private package from a private index. This only affects use of the --extra-index-url option, and exploitation requires that the package does not already exist in the public index (and thus the attacker can put the package there with an arbitrary version number). NOTE: it has been reported that this is intended functionality and the user is responsible for using --extra-index-url securely",
    "debian": {}
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "OR",
        "criterias": [
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.debian.oval:tst:2",
                "comment": "all architecture"
              },
              {
                "test_ref": "oval:org.debian.oval:tst:14737",
                "comment": "python-pip DPKG is earlier than 0"
              }
            ]
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.debian.oval:tst:1",
        "comment": "Debian 11 is installed"
      }
    ]
  }
}
